Makefile

```diff
@@ -0,0 +1,81 @@
+UID = $(shell id -u)
+GID = $(shell id -g)
+docker-exec = docker run --rm -v $(PWD):/usr/app -w /usr/app runroom_samples_bundle $(1)
+
+.PHONY: build halt destroy provision ssh composer-update composer-install composer-normalize phpstan psalm php-cs-fixer phpunit phpunit-coverage rector lint-qa
+
+up:
+	docker build -t runroom_samples_bundle .
+	docker run -d -v $(PWD):/usr/app --name runroom_samples_bundle runroom_samples_bundle
+.PHONY: up
+
+halt:
+	docker stop runroom_samples_bundle
+.PHONY: halt
+
+destroy:
+	docker stop runroom_samples_bundle && docker rm runroom_samples_bundle
+.PHONY: destroy
+
+build: halt destroy
+	docker build --build-arg UID=$(UID) --build-arg GID=$(GID) -t runroom_samples_bundle .
+.PHONY: build
+
+provision: composer-install
+
+down:
+	docker stop runroom_samples_bundle && docker rm runroom_samples_bundle
```

Comment:
I do not like a lot the fact that the container keep running instead of just run while you execute the command, but we can iterate it later...